README.md

```diff
@@ -60,10 +60,10 @@ On Ubuntu >= 18.04 (for example):
 
 ```sh
 sudo apt-get install --yes                                                       \
-            autoconf bison clang-8 cmake curl flex gcc jq libboost-test-dev      \
+            autoconf bison clang-10 cmake curl flex gcc jq libboost-test-dev     \
             libcrypto++-dev libffi-dev libgflags-dev libjemalloc-dev libmpfr-dev \
             libprocps-dev libsecp256k1-dev libssl-dev libtool libyaml-dev        \
-            lld-8 llvm-8-tools make maven netcat-openbsd openjdk-11-jdk          \
+            lld-10 llvm-10-tools make maven netcat-openbsd openjdk-11-jdk        \
             pkg-config python3 python-pygments python-recommonmark               \
             python-sphinx rapidjson-dev time z3 zlib1g-dev
```

Comment:
`python-recommonmark` and `python-sphinx` were removed from the Dockerfile, should the README still say to install them?

Reply:
Yep, I removed them.